lib/typescript/httpclient/endpoints/listTags.ts

```diff
@@ -0,0 +1,28 @@
+import {doFetchFromBackend} from '../api';
+import {Tag, ColorTag} from '../model';
+import {ListTagsResponsePayload} from '../payload';
+
+const colorMapper = (color: string): ColorTag => {
+  const colors = ['BLUE', 'RED', 'GREEN', 'PURPLE'];
+  if (colors.includes(color)) {
+    return `tag-${color.toLowerCase()}` as ColorTag;
+  } else {
+    return 'tag-blue';
+  }
+};
+
+const tagsMapper = (serverTags: Tag[]): Tag[] => {
+  return serverTags.map(t => {
+    return {id: t.id, name: t.name, color: colorMapper(t.color)};
+  });
+};
```

Comment:
the map function here can be inlined (we'd save the return and the the single line would be good enough, not too long)